components/omega/doc/devGuide/MetaData.md

```diff
@@ -0,0 +1,82 @@
+(omega-dev-metadata)=
+
```

Comment:
Entries for both this file and the user guide section need to be added to the top level index.html file in the doc directory

Reply:
added them in index.md